### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ilzosoh**
I would like to update the README file. Please approve this change.

### CODE REVIEW

**Review by ilzoso — CHANGES_REQUESTED** [review]
Not up to the mark... please put in some more effort in this...

**Review by ilzosoh — COMMENTED** [review]
Okay!

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# public
\ No newline at end of file
+# public
+
```
Comment (ilzosoh) [inline README.md:2]: ok!!!

**Review by ilzosoh — COMMENTED** [review]
Done.

**Comment by ilzoso** [comment]
Now that looks good

**Review by ilzoso — APPROVED** [review]

**Review by ilzoso — APPROVED** [review]
Approved!!!

**Pull request merged**